src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colori getRandomColor() {
+        int randomIndex = new Random().nextInt(Colori.values().length);
```

Comment:
The number of enum values is a constant, so it's better to calculate it once and save it into a constant variable. This will improve performance as the `values()` method creates a new array every time it's called.